zap-widget/src/components/SearchBox.tsx

```diff
@@ -0,0 +1,172 @@
+        const packageInfo = await response.json()
+        console.log(packageInfo)
+
+        if (!packageInfo || packageInfo === 'Not Found') {
+          setResult({
+            warn: true,
+            packageName: packageQueryName,
+            hint: '⚠️ Can’t find this package. Typo or doesn’t exist? 🤔',
+          })
+          return
+        } else if (!packageInfo.funding || packageInfo.funding.type !== 'lightning') {
+          setResult({
+            warn: true,
+            packageName: packageQueryName,
+            hint: '⚠️ This package does not accept bitcoin tips yet 🥺',
+          })
+          return
+        } else {
+          const lnAddress = packageInfo.funding.url
+
+          setResult({
+            warn: false,
+            lnAddress: lnAddress.startsWith('lightning:') ? lnAddress.substring(10) : lnAddress,
+            packageName: packageInfo.name,
+            description: packageInfo.description,
+          })
+        }
+      }
+    } catch (err) {
+      console.error('Error fetching package:', err)
+    } finally {
+      setIsLoading(false)
+    }
+  }
+
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen mx-8">
+      <div className="w-full flex items-center justify-center flex-col md:flex-row gap-3">
+        <input
+          value={packageQueryName}
+          onChange={(e) => setPackageQueryName(e.target.value)}
+          placeholder="Search package name..."
+          className="w-full md:w-[576px] h-12 md:h-[72px] rounded-full pl-5 py-2 bg-zap-gradient border border-white/25 text-2xl font-grotesk placeholder-opacity-70"
+        />
+        <button
+          onClick={fetchPackage}
+          disabled={isLoading}
+          className={`hover:invert w-full md:w-[120px] h-12 md:h-[72px] flex items-center justify-center gap-2 rounded-full font-bold text-2xl md:text-3xl transition-all duration-200 ${
+            isLoading ? 'cursor-not-allowed' : 'bg-white text-black '
+          } shadow-md`}
+        >
+          {isLoading && !result ? (
+            <>
+              <span className="animate-spin rounded-full h-5 w-5 border-t-2 border-b-2 hover:border-white"></span>
+            </>
+          ) : (
+            <>
+              {!result && <span>GO</span>}
+              {result && <img src="./cancel.svg" className="w-5 h-5" />}
+            </>
+          )}
+        </button>
+      </div>
+      {/* Result Card */}
+      {result && (
+        <div className="bg-zap-gradient mt-4 p-6 rounded-4xl w-full md:w-[675px]">
+          <div className="flex flex-col gap-3">
+            <h3 className="font-bold text-xl">{result?.packageName}</h3>
+            {/* description */}
+            {result?.warn ? (
+              <p className="text-[#FF6A6A] mt-4">{result?.hint}</p>
+            ) : (
+              <div className="flex flex-col gap-3 text-[#F5F5F5] font-normal ">
+                <p>{result?.description}</p>
```

Comment:
This description and other parts in this result card should actually have a grayish color (notice figma, it's not a simple change of font weight but colors too)